Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Default ignored files\r\n/shelf/\r\n/workspace.xml\r\n# Editor-based HTTP Client requests\r\n/httpRequests/\r\n# Environment-dependent path to Maven home directory\r\n/mavenHomeManager.xml\r\n# Datasource local storage ignored files\r\n/dataSources/\r\n/target/\r\n/dataSources.local.xml\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
--- a/.idea/.gitignore	(revision 9f9668c9aa30b4c078280a8e26af6d3540679bbd)
+++ b/.idea/.gitignore	(date 1753043723889)
@@ -7,5 +7,5 @@
 /mavenHomeManager.xml
 # Datasource local storage ignored files
 /dataSources/
-/target/
+**/target/
 /dataSources.local.xml
